src/DonationForms/ViewModels/DonationFormViewModel.php

```diff
@@ -212,26 +213,26 @@ public function exports(): array
         $formDesign = $this->getFormDesign($this->designId());
 
         return [
-            'donateUrl' => $donateUrl,
-            'validateUrl' => $validateUrl,
-            'authUrl' => $authUrl,
+            'donateUrl'            => $donateUrl,
```

Comment:
something weird going on with the formatting here

Reply:
<img width="1664" alt="Screen Shot 2024-03-08 at 9 53 42 AM" src="https://github.com/impress-org/givewp/assets/75056371/fe4244c5-4646-4f09-afc7-246e1d9b38a6">

I have arrows aligned as the default format in my ide.

Reply:
@JoshuaHungDinh yes please

Reply:
Resolved: dc395febde39970bb5a8639e710ec5e2b6611e00